JPF/bad_buildings.sql

```diff
@@ -117,20 +151,22 @@ select
 	hb.lot as lot,
```

Comment:
Where does boroid in the above concat statement come from? Shouldn't it be hb.boroid?